addons/test_mail/tests/test_mail_thread_internals.py

```diff
@@ -199,6 +200,24 @@ def test_cache_invalidation(self):
         self.env['res.partner'].create({'name': 'Actual Partner'})
         self.assertTrue(record.name)
 
+    def test_chatter_message_as_marked_up_value(self):
+        attachment = namedtuple('Attachment', ('fname', 'content', 'info'))
+        attachments = [attachment('test_image.jpeg','test',{'encoding': None, 'cid': 'ii_lps7a8sm0'})]
```

Comment:
Actually let's do this instead if possible, avoids importing this just for that. And it's more in line with the regular use.
```suggestion
        attachments = [self.env['mail.thread']._Attachment('test_image.jpeg', 'test', {'encoding': None, 'cid': 'ii_lps7a8sm0'})]
```